microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -13,7 +13,7 @@ toc::[]
 [id="microshift-4-14-about-this-release"]
 == About this release
 // TODO: Update with the relevant information closer to release.
-{product-title} (link:https://access.redhat.com/errata/RHSA-2023:XXXX[RHSA-2023:XXXX]) is now Generally Available. This release uses link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.27.md[Kubernetes 1.27] with CRI-O runtime. New features, changes, and known issues that pertain to {product-title} {product-version} are included in this topic.
+{product-title} was previously introduced as Technology Preview in {product-title} 4.13 and is now Generally Available with {product-title} 4.14. This release uses link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.27.md[Kubernetes 1.27] with CRI-O runtime. New features, changes, and known issues that pertain to {product-title} {product-version} are included in this topic.
```

Comment:
We also could leave that it was previously introduced as TP in 4.13 since that statement is true, but also add a note that says something like:

"Support for {product-title} 4.13 is planned to end in a future {product-title} release. You should plan to update to the latest version of {product-title}."